Targets/JavaScriptCore/fuzzbuild.sh

```diff
@@ -18,6 +18,10 @@ export WEBKIT_OUTPUTDIR=FuzzBuild
 
 if [ "$(uname)" == "Linux" ]; then
     ./Tools/Scripts/build-jsc --jsc-only --debug --cmakeargs="-DENABLE_STATIC_JSC=ON -DCMAKE_C_COMPILER='/usr/bin/clang' -DCMAKE_CXX_COMPILER='/usr/bin/clang++' -DCMAKE_CXX_FLAGS='-fsanitize-coverage=trace-pc-guard -O3 -lrt'"
+elif [ "$(uname)" == "Darwin" ]; then
```

Comment:
While you're here, is our [jsc patch](https://github.com/googleprojectzero/fuzzilli/tree/main/Targets/JavaScriptCore/Patches) still necessary? I believe JSC now has native support for Fuzzilli, so maybe we can drop that patch entirely? In that case, please also update the [README](https://github.com/googleprojectzero/fuzzilli/blob/main/Targets/JavaScriptCore/README.md)

Reply:
I dont find that until you tell me. I just follow with the old one.   XD 
Yes we should just update the README for the workflow of the build and use for fuzzilli

But for the settings on Darwin, I think we still need to keep this shell with some args be set by `set-webkit-configuration`